### PR TITLE
debug: Add missing debug logs for LE Meta Events

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This adds debug logs for LE Meta Events.
Those are not too heavy to limit the console output.

@sjanc @rymanluk

### CODE REVIEW

File: nimble/host/src/ble_hs_dbg.c [hunk 1]
```diff
@@ -149,6 +149,184 @@ ble_hs_dbg_le_event_disp(uint8_t subev, uint8_t len, uint8_t *evdata)
...
+            break;
+        }
+
+        BLE_HS_LOG(DEBUG, "LE Extended Advertising Report len=%u num=0x%02x ",
+                   len, data->num_reports);
+
+        for (i = 0; i < data->num_reports; i++) {
+            BLE_HS_LOG(DEBUG, "[%d]={evttype=0x%04x advlen=%u}\n",
+                       i, le16toh(params->evt_type), params->adv_data_len);
+            params += 1;
```
Comment (sjanc) [inline nimble/host/src/ble_hs_dbg.c]: you need to adjust this with adv_data_len so params +=1 is not correct here.

Comment (MariuszSkamra) [reply nimble/host/src/ble_hs_dbg.c]: done

**Comment by MariuszSkamra** [comment]
* Added conditional preprocessor directives to exclude Extended Advertising related events if BLE_EXT_ADV is not set

File: nimble/host/src/ble_hs_dbg.c [hunk 2]
```diff
@@ -149,6 +149,188 @@ ble_hs_dbg_le_event_disp(uint8_t subev, uint8_t len, uint8_t *evdata)
...
+        struct hci_le_subev_direct_adv_rpt_param *params = data->params;
+
+        if (len < sizeof(*data) ||
+                len < sizeof(*data) + data->num_reports * sizeof(*params)) {
+            BLE_HS_LOG(DEBUG, "Corrupted LE Directed Advertising Report "
+                       "len=%u\n", len);
+            break;
+        }
+
+        BLE_HS_LOG(DEBUG, "LE Extended Advertising Report len=%u "
```
Comment (sjanc) [inline nimble/host/src/ble_hs_dbg.c]: didn't notice that before,  this message should be fixed :)

Comment (MariuszSkamra) [reply nimble/host/src/ble_hs_dbg.c]: good catch!

**Pull request merged**